conda-recipe/vcdat/meta.yaml

```diff
@@ -20,4 +20,4 @@ source:
   git_rev: v0.0.1
```

Comment:
either that or we create a `0.0.2` tag point on current master. It's better to have it match.